src/fuse_reduce.cpp

```diff
@@ -65,18 +65,10 @@ struct fused_reduce
                return shapes.at(name).lens() == input.lens();
            }))
             MIGRAPHX_THROW("Dimenstion does not match the submodule.");
-        const auto& s = inputs.at(0);
-        auto lens     = s.lens();
-        if(lens != sm->get_output_shapes().front().lens())
-        {
-            for(const auto& axis : axes)
-            {
-                lens[axis] = 1;
-            }
```

Comment:
Are we not concerned about this case anymore then?

Reply:
This was pointless because the output shape already computes the dimensions.